integration/v3_grpc_test.go

```diff
@@ -636,6 +634,68 @@ func TestV3WatchMultipleEventsFromCurrentRevision(t *testing.T) {
 	clus.Terminate(t)
 }
 
+// TestV3WatchMultipleEventsFromUnsynced tests Watch APIs from unsynced watchers.
+func TestV3WatchMultipleEventsFromUnsynced(t *testing.T) {
```

Comment:
This is really similar to `TestV3WatchMultipleEventsFromCurrentRevision`. Can the insert logic be unified to use only txns or multiple PUTs? If so, there can be a helper function `testWatchMultipleEvents(t *testing.T, rev int64)` that handles both cases without so much code duplication.